src/partials/how-it-works.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,100..1000&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/2.0.0/modern-normalize.min.css"
+      integrity="sha512-4xo8blKMVCiXpTaLzQSLSw3KFOVPWhm/TRtuPVc4WG6kUgjH6J03IBuG7JZPkcWMxJ5huwaBpOpnwYElP/m6wg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheets" type="text/css" href="/src/sass/how-it-works.scss" />
+    <title>how it works</title>
+  </head>
+  <body>
+    <section class="hiw">
+      <div class="hiw-order">
+        <h2 class="hiw-title">To order your fruit basket, simply follow these easy steps</h2>
+        <p class="hiw-paragraf">
+          Our baskets are assembled with care and delivered straight to your doorstep, so you can
+          enjoy the taste of fresh fruit without ever leaving your home. Whether you're looking for
+          a healthy snack or a thoughtful gift, our fruit baskets are the perfect choice.
+        </p>
+        <ul class="hiw-list">
+          <li class="hiw-list-item">
+            <div class="hiw-first-step"></div>
+          </li>
+          <li class="hiw-list-item">
+            <div class="hiw-second-step"></div>
+          </li>
+          <li class="hiw-list-item">
+            <div class="hiw-third-step"></div>
+          </li>
+        </ul>
```

Comment:
Tutaj dodaj img gdzie dodasz gif. A to niepotrzebne: <div class="hiw-gif"></div>